spec/operators/publish-spec.ts

```diff
@@ -175,7 +175,7 @@ describe('publish operator', () => {
     it('should disconnect when last subscriber unsubscribes', () => {
       const source =     cold(   '-1-2-3----4-|');
       const sourceSubs =      '   ^        !   ';
-      const replayed = source.publish().refCount();
+      const replayed = (source.pipe(publish()) as ConnectableObservable<string>).refCount();
```

Comment:
Is there are particular reason for not using the `refCount` operator?

Reply:
I just missed it, honestly. It's a bit of a special case, as you can see.